solr-ingest/src/main/java/edu/unc/lib/dl/data/ingest/solr/filter/SetDatastreamFilter.java

```diff
@@ -57,7 +64,11 @@ public void filter(DocumentIndexingPackage dip) throws IndexingException {
         }
 
         boolean ownedByOtherObject = contentObj instanceof WorkObject;
+        // Retrieve list of datastreams preserved with this object
         List<Datastream> datastreams = getDatastreams(fileObj, ownedByOtherObject);
+        // Retrieve list of derivatives associated with the object
```

Comment:
i'm finding the difference between these two retrievals of derivatives to be very subtle. what does "preserved with this object" mean vs. "associated with this object"?

Reply:
the key word is the preceding one, "datastreams" vs "derivatives". I'll switch to the same verb to draw attention there.